packages/contentful--create-contentful-app/src/template.ts

```diff
@@ -48,28 +48,21 @@ async function clone(source: string, destination: string) {
       // to avoid confusing people.
       throw new Error('Destination directory is not empty.');
     }
-    throw e
+    throw e;
   }
 }
 
 function validate(destination: string): void {
```

Comment:
Can't we remove the whole function?

Reply:
I think it might be helpful to validate if it holds a valid `package.json`. wdyt?

Reply:
We will remove that part then as soon as we have an example with both frontend & backend in different folders

Reply:
I guess it depends on the template we build the, but all of the apps we built with frontend&backend have a package.json at root level, so probably this template will also have one

I just think we could prevent some gotchas when people are building custom templates